R/pkg/R/mllib.R

```diff
@@ -694,8 +694,8 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' }
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
-          function(data, blockSize = 128, layers = c(3, 5, 2), solver = "l-bfgs", maxIter = 100,
-                   tol = 0.5, stepSize = 1, seed = 1) {
+          function(data, blockSize = 128, layers, solver = "l-bfgs", maxIter = 100,
```

Comment:
all right, I will add `layers` parameter validation check and move this parameter to front. thanks!

and, the `layers` parameters must be set because it determine the structure of this classifier.
if we do not set `layers` or set it as a null list `c()`, training will throw exception...
you can check the  `MultilayerPerceptronClassifierSuite` input validation test.